examples/gke/manager.yaml

```diff
@@ -0,0 +1,275 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: scylla-manager-system
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: scylla-manager-cluster-member
+  namespace: scylla-manager-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: scylla-manager-cluster-member
+  namespace: scylla-manager-system
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  verbs:
+  - get
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
```

Comment:
I see a lot of these verbs being duplicated. Is there any way to avoid duplicating the same text in many yaml files?

Reply:
This file is autogenerated, base is in `config/manager/rbac/role.yaml`. We have an issue to extract these common entities to single place: #173

Reply:
It is nice that it is autogenerated. It would be even nicer if it was not in the repository.

These things (copy and paste, autogenerated code) add up. I asked git to produce an archive of the master branch of both operator and seastar. The operator archive is 50 MiB. The Seastar one is 4.5 MiB. I have no doubt the seastar one would be a lot bigger if it included the auto generated files and dependencies (like boost).

Even scylla is "only" 47 MiB.

Reply:
I don't like this copy pasting too. But removing it is not trivial. Although It's planned for next release.
For autogenerated files, I refered to it in other comment.